beast/fitting/fit_metrics/kernels.py

```diff
@@ -123,7 +124,7 @@ def __init__(self, shape, h = 1.0, domain = None, norm = None):
             norm = 1.0
         self._normconst = norm
         self.domain = domain
-        if callable(shape):
+        if isinstance(shape, collections.Callable):
```

Comment:
We don't use it.  This was part of previous ways of interpolation - at least that is my memory of this.  We could remove this, but should do this with a different issue/pull request.  Removing old code we are not using is on the to do list and there is an issue for this.